src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -31,6 +31,13 @@ val RsFunction.isMethod: Boolean get() = hasSelfParameters && owner.isImplOrTrai
 val RsFunction.isTest: Boolean
     get() = queryAttributes.isTest
 
+val RsFunction.isMain: Boolean get() {
```

Comment:
Maybe also provide a new function to detect the `#![no_main]` attribute?

Reply:
It's a good idea. Thanks!